### DISCUSSION TITLE
Automerge: Disabled due to failing status checks.

### DISCUSSION THREAD

**Discussion by northern-mindset** [renovatebot/renovate#10004] [Request Help] [answered]
A [PR](https://github.com/PureBhaktiArchive/audioseva/pull/649) at first failed passing checks, but then it was rebased and now it passes all the checks.

Why it still says the following?

> Automerge: Disabled due to failing status checks.

![image](https://user-images.githubusercontent.com/6065071/118362955-3c97ea00-b5bc-11eb-9655-74e0d205e8cc.png)

**Comment by viceice** [comment, marked as answer]
Renovate will automerge after a second run one hour after the new push. Renovate reads the state before it pushes, so the pr description is outdated until renovate runs again.

**Reply by northern-mindset** [reply]
Great, I see. Thanks for prompt reply.
